.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix = ""
```

Comment:
I have not seen this before, does this remove the`v` from releases?

Reply:
> does this remove the `v` from releases?

Yep, it's what we use in the main repo too: https://github.com/stylelint/stylelint/issues/3499#issuecomment-484461930